packages/devp2p/tsconfig.json

```diff
@@ -5,10 +5,10 @@
     "test/**/*.ts",
     "examples/**/*.ts",
     "scripts/**/*.ts",
-    "examples/simple.cts",
```

Comment:
Why did these change? Were these examples not being included before?

Reply:
We just forgot to update the extension when we migrated over to ESM for our base dev environment.  This is just miscellaneous cleanup.